backend/config/config.py

```diff
@@ -95,10 +96,40 @@ def _load_config(path=None):
 
     return config
 
-
+# TODO: Set global and Sentry log level independently
 CONFIG = {
     'geocoding_enabled': True,
     'geocoder': 'oxcoder',
     'measurer': 'haversine',
     'number_of_adequacy_processors': 8,
+    'logging': {
```

Comment:
I kinda like that, should we do the same for C2Q?

Reply:
idk. it's a python-specific thing dictConfig